src/test/java/uk/gov/hmcts/reform/iacasedocumentsapi/domain/entities/ccd/field/DatesToAvoidTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.iacasedocumentsapi.domain.entities.ccd.field;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class DatesToAvoidTest {
+
+    private SimpleDateFormat format = new SimpleDateFormat("yyyy-MM-dd");
+    private final Date dateToAvoid = format.parse("2019-11-29");
```

Comment:
same as the above. Date against LocalDate

Reply:
Replaced with LocalDate